query-planner/src/lib.rs

```diff
@@ -35,9 +36,9 @@ impl<'s> QueryPlanner<'s> {
         QueryPlanner { schema }
     }
 
-    pub fn plan(&self, query: &str) -> Result<QueryPlan> {
+    pub fn plan(&self, query: &str, auto_fragmentization: bool) -> Result<QueryPlan> {
         let query = parse_query(query).expect("failed parsing query");
-        build_query_plan(&self.schema, &query)
+        build_query_plan(&self.schema, &query, auto_fragmentization)
```

Comment:
I feel pretty ok about query and schema as positional parameters, but we may have other options / tweaks for query planning in the future. This might only be an appropriate JS suggestion (so please correct me if it is) but I'd recommend this 3rd arg be an options struct.

Reply:
Oh sure, good call. We should probably make its construction a builder pattern with default value so that future changes are backwards compatible.